### PR TITLE
adding tutorials for resnet50 and lenet5

### PR DESCRIPTION
**Pull request by lyd911**

### CODE REVIEW

File: tinyms/serving/client/client.py [hunk 1]
```diff
@@ -69,4 +70,13 @@ def predict(img_path, servable_name, dataset_name="mnist"):
     elif res_body['status'] != 0:
```
Comment (leonwanghui) [inline tinyms/serving/client/client.py:70]: Please add the strategy as the parameter of `predict` method and the default one is `TOP1_CLASS`.

Comment (lyd911) [reply tinyms/serving/client/client.py:73]: done, strategy is added in predict, and the output format and score are both rendered

**Review by leonwanghui — APPROVED** [review]
LGTM

**Review by leonwanghui — APPROVED** [review]
LGTM

**Review by huxiaoman7 — APPROVED** [review]

**Pull request merged**